src/App.jsx

```diff
@@ -123,9 +123,9 @@ export async function decodeFSH(encodedFSH) {
   } else {
     const promisedURL = await expandLink(encodedFSH);
 
-    // Removes the encoded data from the end of the url, starting at index 38
+    // Removes the encoded data from the end of the url, starting at index 40
     const sliced64 = promisedURL.long_url.slice(40);
-    if (!promisedURL.long_url.includes('https://fshschool.org/FSHOnline/#/share/') || sliced64.length === 0) {
+    if (!promisedURL.long_url.includes('https://fshonline.fshschool.org/#/share/') || sliced64.length === 0) {
```

Comment:
I have a question about this line. If someone tries to visit an old FSH Online share link, the `promisedURL` will be the returned results from Bitly. So for old share links that were made before the repos move, I think the `promisedURL.long_url` will be `https://fshschool.org/FSHOnline/#/share/...` when it gets here. Is there some redirect that happens somewhere within Bitly's resolution of the long URL it knows about that will let the `promisedURL.long_url` value to be udpated to be in the `fshonline.fshschool.org` format? If not, should we just allow either format here so that we properly decode the URL and load in the shared FSH?

Right now, when I tried to load an old share link, it doesn't work because the long URL is in the wrong format and the text doesn't load, which is why I'm wondering about a sneaky redirect that might not be happening yet.

Reply:
Oh. That's an excellent point. I think you're right. I was thinking (for some reason) that the bitly just had the encoded FSH. I forgot it was still essentially a URL. So we'll need to account for either URL in the code.

Reply:
Fixed in fb0c1c2.